serverless/lint-staged.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Seems like this config is overlapping the config on `.husky/lint-staged.config.json` maybe we need to setup single rules for the entire project although with the root `.eslintrc` file. The same applies to some packages that are installed for `serverless` project that seems like it should be over the root project like the packages of `@commitlint`.